addons/interaction/XEH_postInit.sqf

```diff
@@ -108,3 +108,11 @@ GVAR(isOpeningDoor) = false;
 
 ["isNotSwimming", {!underwater (_this select 0)}] call EFUNC(common,addCanInteractWithCondition);
 ["isNotOnLadder", {getNumber (configFile >> "CfgMovesMaleSdr" >> "States" >> animationState (_this select 0) >> "ACE_isLadder") != 1}] call EFUNC(common,addCanInteractWithCondition);
+
+["ace_settingsInitialized", {
+    if (GVAR(disableNegativeRating)) then {
+        ACE_player addEventHandler ["HandleRating", {
```

Comment:
I think either this should be `player` (as you'd want it to apply to the actual player and not something they're controlling) or you should be using CBA's `"unit"` [player event](https://github.com/CBATeam/CBA_A3/wiki/Player-Events) to add the same rating handling whenever the player character changes.